TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/200_Assets/Scripts/SwitchRenderingMode.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
```

Comment:
This script changes lighting mode in the new test scenes to prevent me having to create a separate renderer for each test scene